### ISSUE TITLE
[BUG] Location is quoted

### ISSUE THREAD

**Issue by geoffjukes** [saltstack-formulas/snmp-formula#38]
Line 19 and 20 in `files/snmpd.conf.minimal` forcefully adds quotes around the strings:
(edit: ditto for lines 186 & 187 of `files/snmpd.conf`)
```
syslocation "{{ conf.get('location', 'Unknown (add saltstack pillar)') }}"
syscontact "{{ conf.get('syscontact', 'Root <root@localhost> (add saltstack pillar)') }}"
```
This results in a Location string of `"Someplace"` as opposed to the desired `Someplace`. Is there a reason for this?

When setting the location in other SNMP devices, the Location string is unquoted, and so this creates 2 separate grouping in our monitoring system.

**Comment by myii** [comment]
@geoffjukes Thanks for the report.  Both of these lines were introduced around 5 years ago:

* [snmpd.conf](https://github.com/saltstack-formulas/snmp-formula/blame/420372a66b5ec52e746f4cb4e4971beb9a664f96/snmp/files/snmpd.conf#L186)
* [snmpd.conf.minimal](https://github.com/saltstack-formulas/snmp-formula/blame/fe0e3e96dfac010a6b592eb44638a3c04c34d36e/snmp/files/snmpd.conf.minimal#L19)

Looking at the [manpage for snmpd.examples](https://www.systutorials.com/docs/linux/man/5-snmpd.examples/), it appears that these need to be updated to remove the quotes.

Would you care to tackle this in a PR?

CC: @alxwr.

**Comment by myii** [comment]
@geoffjukes ... and _now_ I realise you've already submitted the fix in #39!  Thanks for that and let's continue the discussions there instead.

**Comment by ixs** [comment]
Closing the bug, the PR #39 got merged. Again, thanks for your report and the PR @geoffjukes.
